### PR TITLE
Change RefreshIDToken to SessionRefresh, fixes #200

### LINKED ISSUE #200: Change variable name `mozilla_django_oidc.middleware.RefreshIDToken` to something else
Kind of a nitpick, however, `RefreshIDToken` is a confusing name because refresh tokens are a very specific thing: non-expiring access tokens.
In this case, this is the opposite, its a token that expires and needs periodic refresh.

### PR DESCRIPTION
**Pull request by willkg**
The middleware has nothing to do with refresh tokens, so switching the
name makes this less confusing.

### CODE REVIEW

**Comment by willkg** [comment]
This is a backwards incompatible API change, so I upped the version to 1.0.0.

File: HISTORY.rst [hunk 1]
```diff
@@ -3,14 +3,15 @@
 History
 -------
 
-0.7.0 (unreleased)
+1.0.0 (unreleased)
```
Comment (johngian) [inline HISTORY.rst]: ``0.7.0`` is not released so we don't need to bump another version since its already backwards incompatible.

**Comment by johngian** [comment]
Code looks OK. I am not sure about the version bump though, why do we need to change it to `1.0.0` ?

File: HISTORY.rst (hunk 1 above)
Comment (willkg) [reply HISTORY.rst]: If we're doing semver, then this is a backwards incompatible API change which requires a major version bump.

**Pull request merged**